language/analysis/src/main/java/com/google/cloud/language/samples/Analyze.java

```diff
@@ -188,41 +161,30 @@ public Analyze(CloudNaturalLanguage languageApi) {
    */
   public List<Entity> analyzeEntities(String text) throws IOException {
     AnalyzeEntitiesRequest request =
-        new AnalyzeEntitiesRequest()
-            .setDocument(new Document().setContent(text).setType("PLAIN_TEXT"))
-            .setEncodingType("UTF16");
-    CloudNaturalLanguage.Documents.AnalyzeEntities analyze =
-        languageApi.documents().analyzeEntities(request);
-
-    AnalyzeEntitiesResponse response = analyze.execute();
-    return response.getEntities();
+        AnalyzeEntitiesRequest.newBuilder()
+            .setDocument(Document.newBuilder().setContent(text).setType(Type.PLAIN_TEXT))
+            .setEncodingType(EncodingType.UTF16).build();
+    AnalyzeEntitiesResponse response = languageApi.analyzeEntities(request);
+    return response.getEntitiesList();
   }
 
   /**
    * Gets {@link Sentiment} from the string {@code text}.
    */
   public Sentiment analyzeSentiment(String text) throws IOException {
-    AnalyzeSentimentRequest request =
-        new AnalyzeSentimentRequest()
-            .setDocument(new Document().setContent(text).setType("PLAIN_TEXT"));
-    CloudNaturalLanguage.Documents.AnalyzeSentiment analyze =
-        languageApi.documents().analyzeSentiment(request);
-
-    AnalyzeSentimentResponse response = analyze.execute();
+    AnalyzeSentimentResponse response = languageApi.analyzeSentiment(
+        Document.newBuilder().setContent(text).setType(Type.PLAIN_TEXT).build());
     return response.getDocumentSentiment();
   }
 
   /**
    * Gets {@link Token}s from the string {@code text}.
    */
   public List<Token> analyzeSyntax(String text) throws IOException {
-    AnalyzeSyntaxRequest request =
-        new AnalyzeSyntaxRequest()
-            .setDocument(new Document().setContent(text).setType("PLAIN_TEXT"))
-            .setEncodingType("UTF16");
-    CloudNaturalLanguage.Documents.AnalyzeSyntax analyze =
-        languageApi.documents().analyzeSyntax(request);
-    AnalyzeSyntaxResponse response = analyze.execute();
-    return response.getTokens();
+    AnalyzeSyntaxRequest request = AnalyzeSyntaxRequest.newBuilder()
+            .setDocument(Document.newBuilder().setContent(text).setType(Type.PLAIN_TEXT).build())
```

Comment:
Revised this line and adjacent lines to build document separate from request - makes all the lines a bit less lengthy.